src/compiler/msvc.rs

```diff
@@ -328,12 +390,13 @@ pub fn parse_arguments(arguments: &[OsString]) -> CompilerArguments<ParsedArgume
             }
         };
     }
+
     CompilerArguments::Ok(ParsedArguments {
         input: input.into(),
         language: language,
         depfile: depfile.map(|d| d.into()),
         outputs: outputs,
-        preprocessor_args: vec!(),
+        preprocessor_args: preprocessor_args,
```

Comment:
I was a little unclear why all of these used to go in common args - it's pessimistic so wouldn't be a problem, and I couldn't see anything in the git history indicating why it's like this.

Reply:
I couldn't figure this out either, TBH. This is definitely better, though.